cloudcomposerdiff/requirements.txt

```diff
@@ -0,0 +1,63 @@
+#
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
I discovered pip-compile through this article: https://realpython.com/pypi-publish-python-package/

I will try to answer your questions here:

1. yes, implicit & explicit dependencies
2. May be easier to re-jig things on my end to fit y our Med renovate tooling
3. Please get some other opinions from python folks with more packaging experience. happy to pin versions based on their advice
4. yes, it'd be great if this were automated. I often forget to do this at the moment because it's a manual process of me executing the command in the CONTRIBUTING.md guide

Leaving this comment open to capture advice of you & your colleagues on how to proceed.